### PR TITLE
Add setting to change the toast lifetime

### PR DESCRIPTION
**Pull request by SamuelWei**
## Type (Highlight the corresponding type)
- Bugfix
- **Feature**
- Documentation
- Refactoring (e.g. Style updates, Test implementation, etc.)
- Other (please describe):

## Checklist
- [x] Code updated to current develop branch head
- [x] Passes CI checks
- [ ] Is a part of an issue
- [x] Tests added for the bugfix or newly implemented feature, describe below why if not
- [x] Changelog is updated
- [x] Documentation of code and features exists

## Changes

- Add setting to change the lifetime of toast messages
- Fixed issue in db:upgrade command causing issues with settings added after the db upgrade

### CODE REVIEW

**Comment by SamuelWei** [comment]
@danielmachill @Sabr1n4W If you both approve, you can merge next week

File: lang/en/settings.php [hunk 1]
```diff
@@ -128,6 +128,11 @@
         'three_month' => '3 Months (90 Days)',
         'tile_description' => 'Controls system-wide settings such as logo, maintenance banner and page sizes.',
         'title' => 'Application',
+        'toast_lifetime' => [
+            'custom' => 'Custom',
+            'description' => 'Time in seconds that a notification is displayed',
```
Comment (Sabr1n4W) [inline lang/en/settings.php:133]: Consider changing to 'Time in seconds for which a notification is displayed'

File: lang/de/settings.php [hunk 2]
```diff
@@ -128,6 +128,11 @@
         'three_month' => '3 Monate (90 Tage)',
         'tile_description' => 'Regelt systemweitere Einstellungen wie Logo, Wartungs-Banner und Seitengrößen.',
         'title' => 'Konfiguration',
+        'toast_lifetime' => [
+            'custom' => 'Benutzerdefiniert',
+            'description' => 'Zeit in Sekunden, die eine Benachrichtigung angezeigt wird',
```
Comment (Sabr1n4W) [inline lang/de/settings.php:133]: Consider changing to 'Zeit in Sekunden, für die eine Benachrichtigung angezeigt wird'

**Review by Sabr1n4W — COMMENTED** [review]
Add translation for the error messages? (toast lifetime)

File: tests/Unit/Console/UpgradeDatabaseTest.php [hunk 3]
```diff
@@ -52,10 +52,8 @@ public function testV2Database()
         $this->artisan('migrate:fresh');
         $migrations_v4 = DB::table('migrations')->pluck('migration')->toArray();
 
-        // Check migration table after upgrade starts with the same as a fresh v4 database
-        foreach ($migrations_upgrade as $key => $migration) {
-            $this->assertEquals($migration, $migrations_v4[$key]);
-        }
+        // Check migration table after upgrade looks like a fresh v4 database
```
Comment (danielmachill) [inline tests/Unit/Console/UpgradeDatabaseTest.php:55]: Consider adding a clause:'Check the migration table after the upgrade to see if it looks like a new v4 database.'

**Comment by SamuelWei** [comment]
> Add translation for error messages? (toast lifetime)

Yeah, good point. Fixed

**Pull request merged**